app/main.py

```diff
@@ -1,2 +1,49 @@
+INITIAL_CAPACITY = 8
+LOAD_FACTOR = 2 / 3
+RESIZE_STRATEGY = 2
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self
+    ) -> None:
+        self.length = 0
+        self.hash_table = [[] for _ in range(INITIAL_CAPACITY)]
+
+    def __len__(self) -> int:
+        return self.length
+
+    def resize(self) -> None:
+        old_table = self.hash_table
+        self.hash_table = [
+            [] for _ in range(len(self.hash_table) * RESIZE_STRATEGY)
+        ]
+        self.length = 0
+        for node_list in old_table:
+            for node in node_list:
+                self.__setitem__(node[0], node[2])
```

Comment:
In the `resize` method, when re-inserting items, the value should be `node[1]` instead of `node[2]`. This is because `node[1]` is the hash value, and `node[2]` is the value itself.